en/operations/admin-procedures.html

```diff
@@ -121,6 +121,217 @@ <h2 id="status-pages">Status pages</h2>
+                                   '{ input artist | exact | summary artist | attribute artist | index artist; }'\n"
+    }
+  ],
+  "tenant": "default",
+  "url": "http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default",
+  "message": "Session 3 for tenant 'default' prepared and activated.",
+  "configChangeActions": {
+    "restart": [],
+    "refeed": [],
+    "reindex": [
+      {
+        "name": "indexing-change",
+        "documentType": "music",
+        "clusterName": "music",
+        "messages": [
+          "Document type 'music': Field 'artist' changed: add index aspect, indexing script: '{ input artist | summary artist | attribute artist; }' -> '{ input artist | exact | summary artist | attribute artist | index artist; }'"
+        ],
+        "services": [
+          {
+            "serviceName": "searchnode",
+            "serviceType": "searchnode",
+            "configId": "music/search/cluster.music/0",
+            "hostName": "vespa-container"
+          }
+        ]
+      }
+    ]
+  }
+}
+</pre>
+<p>
+  Trigger reindexing, deploy again, wait until done
+  (look for the log message / use the <code>reindexing</code> endpoint).
+  Dumping the index structures now show artist both in index and attribute.
+  Verify the query still works:
+</p>
+<pre>
+$ curl http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default/reindex
+
+{
+  "generation": 3,
+  "applicationPackageFileReference": "",
+  "modelVersions": ["7.423.37"]
+}
+
+$ curl http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default/reindexing
+{
+  "enabled": true,
+  "clusters": {
+    "music": {
+      "pending": {
+        "music": 3
+      },
+      "ready": {
+        "music": {}
+      }
+    }
+  }
+}
+
+$ (cd src/main/application && zip -r - .) |   curl --header Content-Type:application/zip --data-binary @- \
+  localhost:19071/application/v2/tenant/default/prepareandactivate
+
+$ curl http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default/reindexing
+{
+  "enabled": true,
+  "clusters": {
+    "music": {
+      "pending": {},
+      "ready": {
+        "music": {
+          "readyMillis": 1628241149710,
+          "startedMillis": 1628241179977,
+          "endedMillis": 1628241180525,
+          "state": "successful"
+        }
+      }
+    }
+  }
+}
+
+$ docker exec vespa /usr/bin/sh -c '/opt/vespa/bin/vespa-logfmt | grep Reindexer'
+[2021-08-06 09:13:00.524] INFO    : container-clustercontroller Container.ai.vespa.reindexing.Reindexer
+                                    Completed reindexing of datatype music (code: 1412693671) after PT0.547121S
+
+$ docker exec vespa /usr/bin/sh -c '/opt/vespa/bin/vespa-proton-cmd --local triggerFlush' && \
```

Comment:
What/why is this?

Reply:
I added this to be able to see the effects of the indexing changes - I think I will move this to an appendix instead, for debugging purposes, not really needed here